.github/allowed-labels.yml

```diff
@@ -115,72 +126,30 @@
   color: "00FFFF"
   description: "This work needs input from the reporter"
   from_name: "needs input"
-- name: status/needs-research
+- name: research-spike
   color: "00FFFF"
   description: "This work needs research by the resolver"
-  from_name: "needs research"
+  from_name: "status/needs-research"
+- name: status/on-hold
```

Comment:
This should just be called `status/do-not-merge`.

Reply:
I think that's a good idea, but I'll deal with it in a follow up PR if we feel it's a good change.